test/unit/utils/json-utils.spec.js

```diff
@@ -0,0 +1,11 @@
+'use strict'
+
+const JsonUtils = require('../../../lib/utils/json-utils')
+
+describe('Utils', () => {
```

Comment:
It would be nice to add one test that covers circular structure issue.

Reply:
done